zebra-state/src/service/read/block.rs

```diff
@@ -1,14 +1,29 @@
 //! Shared block, header, and transaction reading code.
+//!
+//! In the functions in this module:
+//!
+//! The StateService commits blocks to the finalized state before updating
+//! `chain` or `non_finalized_state` from the latest chains. Then it can
```

Comment:
I still have trouble with `latest chains`.

Does it refer to the most recently updated chain in the `chain_set` in the `NonFinalizedState`?

Reply:
I'll make this change to all these files in my editor.
At the same time, I'll change `StateService` to "block commit task", because that's what it will be after this PR series.

```suggestion
//! `chain` or `non_finalized_state` with a cached copy of the non-finalized chains in `NonFinalizedState.chain_set`. Then it can
```

Reply:
Updated in PR #5134 in commit 22ef6561f8